src/constants.ts

```diff
@@ -1,3 +1,10 @@
 export const ACCESS_TOKEN = 'access';
```

Comment:
I don't think we need these anymore, I know it's not part of this PR but might wanna look into it.